### PR TITLE
Move the client build for selenium to in-repo script

### PR DESCRIPTION
**Pull request by dannon**
It's currently in in the external jenkins job configuration, and I'll remove it form there if this seems to work the same.  This should fix what we were seeing @nsoranzo 

(this build will show a double client build, in the selenium tests output, since I haven't disabled the external invocation of this yet)

### CODE REVIEW

File: .ci/jenkins/selenium/run_tests.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ fi
 mkdir -p "$GALAXY_TEST_ERRORS_DIRECTORY"
 mkdir -p "$GALAXY_TEST_SCREENSHOTS_DIRECTORY"
 
+docker run -v `pwd`:`pwd`:rw -w `pwd` -u $UID node /bin/bash -c 'make client-production'
```
Comment (martenson) [inline .ci/jenkins/selenium/run_tests.sh]: The jenkins job has `make client-production-maps` - I am not sure that it makes a difference here.

Comment (dannon) [reply .ci/jenkins/selenium/run_tests.sh]: Yeah, I swapped that thinking we might as well save the little bit of time.  If the selenium test output actually resolves through the sourcemaps, and it's useful, I can definitely swap it back.

Comment (jmchilton) [reply .ci/jenkins/selenium/run_tests.sh]: I believe the sourcemaps do produce better outputs in the tests - such as for the captured console logs.

**Comment by nsoranzo** [comment]
Thanks @dannon !

File: .ci/jenkins/selenium/run_tests.sh (hunk 1 above)
Comment (dannon) [reply .ci/jenkins/selenium/run_tests.sh]: Hrmm, I didn't think they actually resolved through the test output, but I can swap it.  Really, for the best test output, we should just do 'make client', but then it's not testing the production optimizations.

Comment (jmchilton) [reply .ci/jenkins/selenium/run_tests.sh]: It is possible they weren't - I didn't have evidence just intuition and vague recollections.

**Comment by dannon** [comment]
It looks like the relocated build works well.  When this is merged, we should remove the one from jenkins (and if whoever merges this doesn't have access, that's fine, I'll get it).

**Comment by martenson** [comment]
jenkins job's config adjusted

**Pull request merged**